### PR TITLE
Tor: add info page

### PR DESCRIPTION
**Pull request by fabianfrz**
@fichtner this the info page - at the moment it shows the hidden service hostnames.

### CODE REVIEW

**Comment by fabianfrz** [comment]
spectacle ik8964

**Comment by fichtner** [comment]
More or less unrelated but while at it:

screen shot 2017-09-23 at 3 07 54 pm

**Comment by fabianfrz** [comment]
what permission should it have?

File: security/tor/src/opnsense/service/scripts/tor/setup.sh [hunk 1]
```diff
@@ -3,5 +3,7 @@ mkdir -p /var/db/tor
 mkdir -p /var/log/tor
 mkdir -p /var/run/tor
 
+chmod 751 /var/db/tor
```
Comment (fichtner) [inline security/tor/src/opnsense/service/scripts/tor/setup.sh]: After the warning the directory has 700 permission

**Comment by fabianfrz** [comment]
@fichtner it did not complain about 700 so I have set it to 700 as it is more restrictive

**Pull request merged**